apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -746,8 +748,32 @@
             }
           })
 
+
+
+
           if (this.userKeymap === "vim") {
             const codeMirrorVimInstance = document.querySelector(".CodeMirror").CodeMirror.constructor.Vim
+            const coreTabs = this.$root.$refs.CoreTabs
+            codeMirrorVimInstance.defineEx("write", "w", this.triggerSave)
+            codeMirrorVimInstance.defineEx("quit", "q", this.close)
+            codeMirrorVimInstance.defineEx("x", "x", () => {
+              this.triggerSave()
+              if(this.query.id) {
+                this.close()
+              }
+            })
+            codeMirrorVimInstance.defineEx("tabnext", "tabn", coreTabs.nextTab)
```

Comment:
This stuff looks awesome, but looks like these tab commands will right now only work when the user has a Query Editor in focus, so they won't work on the other tab types.

Can you see an easy way to pop these shortcuts up to the `CoreTabs` keymap computed property? It would require reading the setting to see if the user is in VIM mode or not, but should work pretty well

Reply:
I decided to eliminate the 'tabnext' and 'tabprev' functionalities because they relied on the CodeMirror instance being available. Unfortunately, I couldn't determine how to ensure its availability in other query types, so I opted to remove those features